specs/_features/maxeb_increase/capella.py

```diff
@@ -1575,6 +1582,44 @@ def process_eth1_data_reset(state: BeaconState) -> None:
         state.eth1_data_votes = []
 
 
+def process_pending_balance_deposits(state: BeaconState) -> None:
+    deposit_balance_to_consume = get_validator_churn_limit(state)
+    next_pending_deposit_index = 0
+    for pending_balance_deposit in state.pending_balance_deposits:
+        if state.deposit_validator_balance + deposit_balance_to_consume >= pending_balance_deposit.amount:
+            deposit_balance_to_consume -= pending_balance_deposit.amount - state.deposit_validator_balance
+            state.deposit_validator_balance = Gwei(0)
+            increase_balance(state, pending_balance_deposit.index, pending_balance_deposit.amount)
+            next_pending_deposit_index += 1
+        else:
+            state.deposit_validator_balance += deposit_balance_to_consume
+            break
+    state.pending_balance_deposits = state.pending_balance_deposits[next_pending_deposit_index:]
+
+
+def process_pending_balance_withdrawals(state: BeaconState) -> None:
+    withdrawal_balance_to_consume = get_validator_churn_limit(state)
+    for pending_balance_withdrawal in state.pending_balance_withdrawals:
+        if pending_balance_withdrawal.withdrawable_epoch != FAR_FUTURE_EPOCH:
+            continue
+        if state.withdrawal_validator_balance + withdrawal_balance_to_consume >= pending_balance_withdrawal.amount:
```

Comment:
If `is_exit` the `amount` should rather be `balances[pending_balance_withdrawal.index]`